test/CompileSpec.hs

```diff
@@ -75,8 +75,6 @@ spec = do
             , "double = (x) => (x * 2)"
             , "half :: Num -> Num"
             , "half = (x) => (x / 2)"
-            , "true  = True"
```

Comment:
I understand why it's gone in the snapshot now. Weirdly it seems that it actually does compile to true and false where appropriate.